portfolio/src/main/java/com/google/sps/data/TaskPriority.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+/** Handles range checking, and allows retrieval of the priority value. */
+public final class TaskPriority {
```

Comment:
Consider adding some static variables.

public static int MIN_PRIORITY = 1;
public static int MAX_PRIORITY = 5;

Then you can use them in the constructor.

Reply:
Updated.